docs/data/material/getting-started/templates/sign-in-side/SignInSide.tsx

```diff
@@ -39,90 +101,179 @@ export default function SignInSide() {
-            backgroundSize: 'cover',
-            backgroundPosition: 'center',
-          }}
-        />
-        <Grid item xs={12} sm={8} md={5} component={Paper} elevation={6} square>
-          <Box
-            sx={{
-              my: 8,
-              mx: 4,
-              display: 'flex',
-              flexDirection: 'column',
-              alignItems: 'center',
-            }}
-          >
-            <Avatar sx={{ m: 1, bgcolor: 'secondary.main' }}>
-              <LockOutlinedIcon />
-            </Avatar>
-            <Typography component="h1" variant="h5">
-              Sign in
-            </Typography>
-            <Box component="form" noValidate onSubmit={handleSubmit} sx={{ mt: 1 }}>
-              <TextField
-                margin="normal"
-                required
-                fullWidth
-                id="email"
-                label="Email Address"
-                name="email"
-                autoComplete="email"
-                autoFocus
-              />
-              <TextField
-                margin="normal"
-                required
-                fullWidth
-                name="password"
-                label="Password"
-                type="password"
-                id="password"
-                autoComplete="current-password"
-              />
-              <FormControlLabel
-                control={<Checkbox value="remember" color="primary" />}
-                label="Remember me"
-              />
-              <Button
-                type="submit"
-                fullWidth
-                variant="contained"
-                sx={{ mt: 3, mb: 2 }}
+    <ThemeProvider theme={showCustomTheme ? SignInSideTheme : defaultTheme}>
+      <CssBaseline />
+      <Box
+        sx={{
+          position: 'absolute',
+          width: '100%',
+          height: '100%',
+          top: 0,
+          left: 0,
+        }}
+      >
+        <Grid container component="main" sx={{ height: '100vh' }}>
+          <Grid
+            item
+            xs={false}
+            sm={6}
+            md={8}
+            sx={(theme) => ({
+              backgroundImage:
+                theme.palette.mode === 'light'
+                  ? 'url("/static/images/templates/templates-images/light-background.png")'
+                  : 'url("/static/images/templates/templates-images/dark-background.png")',
+              backgroundRepeat: 'no-repeat',
+              backgroundColor:
+                theme.palette.mode === 'light'
+                  ? theme.palette.grey[50]
+                  : theme.palette.grey[900],
+              backgroundSize: 'cover',
+              backgroundPosition: 'center',
+            })}
+          />
+          <Grid item xs={12} sm={6} md={4}>
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'center',
+                p: 4,
+                pb: 0,
+                gap: 6,
```

Comment:
```suggestion
                gap: 4,
```

Let's make sure this is consistent with the Sign in template, some values here are slightly different